todo_app/data/api.py

```diff
@@ -0,0 +1,70 @@
+from configparser import ConfigParser
+from .boards import *
+from todo_app.config import *
+import os
+import requests
+
+
+class TrelloAPI():
+    TRELLO_URL = 'https://api.trello.com/1'
+    
+    def __init__(self):
+        self.key_and_token = {'key':APIConfig.KEY,'token':APIConfig.TOKEN}
+    
+    def get_boards(self):
+        boards_url = self.TRELLO_URL + '/members/me/boards'
+        key_and_token = self.key_and_token
+        headers = {"Accept": "application/json"}
+        arguments = {'fields': 'name', 'lists': 'open'}
+        response = requests.get(boards_url, params=key_and_token, data=arguments)
+        json_response = response.json()
+        return json_response
+    
+    def get_lists(self): 
+        boardId = "oIJEvych"
```

Comment:
`boardId` and `id_list` etc should be configurable rather than hardcoded in various functions in `api.py`. You will find you need to change this in Module 3 for your end to end test to work.